src/utilities.js

```diff
@@ -0,0 +1,8 @@
+const formatDate = inputDate => {
+  const monthNames = ["January", "February", "March", "April", "May", "June",
+    "July", "August", "September", "October", "November", "December"];
+  let date = new Date(inputDate);
+  return monthNames[date.getMonth()] + ' ' + date.getDate() + ', ' + date.getFullYear();
+}
+
+module.exports = { formatDate };
```

Comment:
Good call writing a helper function to format our dates. This file is a good place for it too.